packages/shared/routes/dashboard/settings/views/advanced/NetworkConfiguration.svelte

```diff
@@ -104,27 +104,29 @@
             </div>
         </div>
     {/if}
-    <HR classes="pb-5 mt-5 justify-center" />
-    <section id="nodeConfiguration">
-        <Text type="h5" classes="mb-3">
-            {localize('views.settings.networkConfiguration.nodeConfiguration.title')}
-        </Text>
-        <Text type="p" secondary classes="mb-5">
-            {localize('views.settings.networkConfiguration.nodeConfiguration.description')}
-        </Text>
-        <Radio
-            value={true}
-            bind:group={clientOptions.automaticNodeSelection}
-            label={localize('views.settings.networkConfiguration.nodeConfiguration.automatic')}
-            subLabel="Connect to official nodes from the IOTA Foundation"
-        />
-        <Radio
-            value={false}
-            bind:group={clientOptions.automaticNodeSelection}
-            label={localize('views.settings.networkConfiguration.nodeConfiguration.manual')}
-            on:change={handleManualNodeSelection}
-        />
-    </section>
+    {#if $activeProfile?.networkType !== NetworkType.PrivateNet}
```

Comment:
Why do we not display this if they are connected to a private network. The user still might want to view the node info or add an additional node.